### ISSUE TITLE
[Improvement] EMI Support

### ISSUE THREAD

**Issue by wavian** [davioliva16/create-aquatic-ambitions#5]
Hey, been enjoying playing around with this mod, title is mostly self-explanatory, add additional support for recipe type in the EMI menus so recipes can be viewed. Thanks!

**Comment by bangetto** [comment]
you ca use JEI to acces the recipes of those mods that don't support EMI yet, although yeah, it would be a nice addition

**Comment by wavian** [comment]
Yeah I may have phrased that a little strangely, but I really like the recipe trees and layout in EMI and would prefer not to have two recipe viewer mods. Most things do actually show up in EMI, just not the Conduit Channeling.

**Comment by bangetto** [comment]
no, EMI hides the JEI menu, and just pulls the information from it so it can display it

**Comment by wavian** [comment]
Thank you, didn't know that however also did not display in EMI even after doing that, it did fix the Create: Enchantment Industry recipes though so it was functioning in some form.

**Comment by davioliva16** [comment]
I needed to add some custom code for JEI (because of the conduit render on JEI), I can try to see REI and EMI are simple, but no promises yet
